src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -220,14 +222,13 @@ public static Material asItem(Material type) {
 	 * @param object Object to convert
 	 * @return ItemStack from slot/itemtype
 	 */
-	@Nullable
-	public static ItemStack asItemStack(Object object) {
-		if (object instanceof ItemType)
-			return ((ItemType) object).getRandom();
-		else if (object instanceof Slot)
-			return ((Slot) object).getItem();
-		else if (object instanceof ItemStack)
-			return ((ItemStack) object);
+	public static @Nullable ItemStack asItemStack(@Nullable Object object) {
+		if (object instanceof ItemType itemType)
+			return itemType.getRandom();
+		else if (object instanceof Slot slot)
+			return slot.getItem();
+		else if (object instanceof ItemStack itemStack)
+			return itemStack;
```

Comment:
Formatting while we're here
```suggestion
		if (object instanceof ItemType itemType) {
			return itemType.getRandom();
		} else if (object instanceof Slot slot) {
			return slot.getItem();
		} else if (object instanceof ItemStack itemStack) {
			return itemStack;
		}
```